src/nncf/quantization/advanced_parameters.py

```diff
@@ -286,6 +290,23 @@ class AdvancedQuantizationParameters:
     # Backend specific parameters
     backend_params: dict[str, Any] = field(default_factory=dict)
 
+    # Backend parameter names
+    COMPRESS_WEIGHTS = "compress_weights"
```

Comment:
```suggestion
    compress_weights = True
```
Please move this line to  the line 272